tasks/util/postcss.ts

```diff
@@ -25,11 +26,12 @@ export function createProcessors(dest: string, cwd = '', dist?: boolean) {
 				const outputPath = path.resolve(dest, path.relative(cwd, cssFileName));
 				const newFilePath = outputPath + '.js';
 				const themeKey = ' _key';
-				json[themeKey] = 'dojo-' + path.basename(outputPath, '.m.css');
+				const packageName = require(pkgUp.sync()).name;
+				json[themeKey] = path.join(packageName, path.basename(outputPath, '.m.css'));
```

Comment:
Will this cause problems across operating systems?

Reply:
Hmm in what way are you thinking? Possibly using `path.join` automatically fixes all the OS issues? To be honest I haven't ever dealt with paths on other OS's

Reply:
Have hardcoded it to forward slash!